src/app/core/core.module.ts

```diff
@@ -1,13 +1,24 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+
 import { AppInterceptor } from './interceptors/app.interceptor';
+import { SharedModule } from '../shared/shared.module';
+import { StoreModule } from '@ngrx/store';
+import { appReducer } from './store/reducers/app.reducer';
+import { EffectsModule } from '@ngrx/effects';
+import { AppEffects } from './store/effects/app.effects';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
 
 @NgModule({
   declarations: [],
   imports: [
     CommonModule,
     HttpClientModule,
+    SharedModule,
+    StoreModule.forRoot(appReducer),
```

Comment:
You have to create Store Module in store folder and import it here to have special module for store with all configurations